cmd/ddltest/column_test.go

```diff
@@ -55,7 +56,7 @@ func (s *TestDDLSuite) checkAddColumn(c *C, rowID int64, defaultVal interface{},
 				// When insert a row with 3 columns, the third column value will be the first column value.
 				newInsertCount++
 			} else {
-				log.Fatalf("[checkAddColumn fail]invalid row: %v", data)
+				log.Fatal(`[checkAddColumn fail]invalid row`, zap.Any(`row`, data))
```

Comment:
```suggestion
				log.Fatal("[checkAddColumn fail]invalid row", zap.Any("row", data))
```

i mean use `"` for all normal strings not the other way around

Reply:
Oooo, i get it finally.

Reply:
Oooo, i get it finally. Please check again :-). thx!